src/graph/unit_graph.cc

```diff
@@ -1492,4 +1527,14 @@ void UnitGraph::Save(dmlc::Stream* fs) const {
   }
 }
 
+UnitGraphPtr UnitGraph::Reverse() const {
+  CSRPtr new_incsr = out_csr_, new_outcsr = in_csr_;
+  COOPtr new_coo = nullptr;
+  if (coo_->defined()) {
```

Comment:
When coo is not defined, at least one of `in_csr_` and `out_csr_` is defined, is my understanding right?

Reply:
Yes